packages/models-library/src/models_library/services.py

```diff
@@ -385,5 +490,35 @@ class ServiceAccessRightsAtDB(ServiceKeyVersion, ServiceGroupAccessRights):
         ..., description="defines the product name", example="osparc"
     )
 
+    @classmethod
+    def create_from(
+        cls, resource: Tuple[Union[str, int], ...], flags: Dict[str, bool]
+    ) -> "ServiceAccessRightsAtDB":
+        return cls(
+            key=resource[0],
+            version=resource[1],
+            gid=resource[2],
+            product_name=resource[3],
+            **flags,
+        )
```

Comment:
I am not sure I understand that function. how as a client shall I know how resource is setup from outside? Also what are flags?
ok I get it more or less, but still find it a bit confusing. so you define a resource, but why call it a resource when we know it is a service? also with the flags, same actually. it kinds of rename something in something else and that is what I find a bit confusing.

Reply:
I agree with you. I removed it from here.